vault/acl.go

```diff
@@ -241,6 +263,24 @@ CHECK:
 		return false, sudo
 	}
 
+	if permissions.MaxWrappingTTL > 0 {
+		if req.WrapInfo == nil || req.WrapInfo.TTL > permissions.MaxWrappingTTL {
```

Comment:
Do we want to enforce wrapping when `max_wrapping_ttl` and/or `min_wrapping_ttl` is set?